### PR TITLE
fix: simplify Panel's implementation

### PR DESCRIPTION
**Pull request by InnaAtanasova**
#### Please provide a brief summary of this pull request.
This is a simplified version of the Panel component with less markup and no service

BREAKING CHANGES: Removed directives: `fd-panel-expand`, `fd-panel-header`, `fd-panel` selector. Before `[fd-panel]`, After:  `fd-panel`

### CODE REVIEW

File: libs/core/src/lib/panel/panel.component.ts [hunk 1]
```diff
@@ -52,42 +47,37 @@ export class PanelComponent implements CssClassBuilder, OnChanges, OnInit, OnDes
     @HostBinding('attr.id')
     id: string = 'fd-panel-' + panelUniqueId++;
 
+    /** Id of the expand button */
+    @Input()
+    expandId: string = 'fd-panel-expand-' + panelExpandUniqueId++;
```
Comment (salarenko) [inline libs/core/src/lib/panel/panel.component.ts:52]: Shouldn't this be applied as `[id]` parameter to expand button?

File: libs/core/src/lib/panel/panel.component.ts [hunk 2]
```diff
@@ -106,14 +96,10 @@ export class PanelComponent implements CssClassBuilder, OnChanges, OnInit, OnDes
...
-            .subscribe(value => {
-                this.expanded = value.isExpanded;
-                this.expandedChange.emit(this.expanded);
-                this._cdRef.detectChanges();
-            });
+    /** Methods that toggles the Panel Content */
+    toggleExpand(): void {
+        this.expanded = !this.expanded;
+        this.expandedChange.emit(this.expanded);
+        this._cdRef.detectChanges();
```
Comment (salarenko) [inline libs/core/src/lib/panel/panel.component.ts:103]: There is no need to manually force detect changes, it will be triggered with `click` event.

Comment (InnaAtanasova) [reply libs/core/src/lib/panel/panel.component.ts:103]: Good point, removed it

File: libs/core/src/lib/panel/panel.component.ts (hunk 1 above)
Comment (InnaAtanasova) [reply libs/core/src/lib/panel/panel.component.ts:52]: Yes, I used it for aria-controls but it's good to have add the id too. Thanks

**Comment by salarenko** [comment]
I think we are good here.
@InnaAtanasova, @droshev?

**Pull request merged**